hack/e2e/run-rp-and-e2e.sh

```diff
@@ -134,20 +139,32 @@ echo "######################################"
 echo "######## Current settings : ##########"
 echo
 echo "LOCATION=$LOCATION"
+echo "AZURE_SUBSCRIPTION_ID=$AZURE_SUBSCRIPTION_ID"
 echo
-echo "COSMOSDB_ACCOUNT=$COSMOSDB_ACCOUNT"
-echo "DATABASE_NAME=$DATABASE_NAME"
-echo "RESOURCEGROUP=$RESOURCEGROUP"
+echo "RP_MODE=$RP_MODE"
+if [ $RP_MODE = "development" ] 
+then 
+    echo
+    echo "COSMOSDB_ACCOUNT=$COSMOSDB_ACCOUNT"
+    echo "DATABASE_NAME=$DATABASE_NAME"
+    echo "RESOURCEGROUP=$RESOURCEGROUP"
+fi
 echo
 echo "CLUSTER=$CLUSTER"
 echo "CLUSTER_RESOURCEGROUP=$CLUSTER_RESOURCEGROUP"
 echo "KUBECONFIG=$KUBECONFIG"
-echo
-echo "PROXY_HOSTNAME=$PROXY_HOSTNAME"
+if [ $RP_MODE = "development" ]
+then 
+    echo
+    echo "PROXY_HOSTNAME=$PROXY_HOSTNAME"
+fi
 echo "######################################"
 
 [ "$LOCATION" ] || ( echo ">> LOCATION is not set please validate your ./secrets/env"; exit 128 )
-[ "$RESOURCEGROUP" ] || ( echo ">> RESOURCEGROUP is not set please validate your ./secrets/env"; exit 128 )
-[ "$PROXY_HOSTNAME" ] || ( echo ">> PROXY_HOSTNAME is not set please validate your ./secrets/env"; exit 128 )
-[ "$COSMOSDB_ACCOUNT" ] || ( echo ">> COSMOSDB_ACCOUNT is not set please validate your ./secrets/env"; exit 128 )
-[ "$DATABASE_NAME" ] || ( echo ">> DATABASE_NAME is not set please validate your ./secrets/env"; exit 128 )
+if [ $RP_MODE = "development" ]
+then 
+    [ "$RESOURCEGROUP" ] || ( echo ">> RESOURCEGROUP is not set please validate your ./secrets/env"; exit 128 )
```

Comment:
Not a big deal, but can we add punctuation to these messages while this is under review again?

> RESOURCEGROUP is not set; please validate your ./secrets/env

or

> RESOURCEGROUP is not set. Please validate your ./secrets/env

Makes it more readable.

Reply:
done